### DISCUSSION TITLE
LDAP AD Bind password in docker compose

### DISCUSSION THREAD

**Discussion by aejazuddinam** [DependencyTrack/dependency-track#1229] [Q&A]
Hi @stevespringett ,
Can you please provide some examples how LDAP Configuration password is bind. My scenario is my LDAP password is stored in a docker secret and calling that docker secret inside the DT Docker compose file is not working, its not recognizing the password. 

I've tried with different scenarios to call Bind password like this once , the docker secret is defined globally and locally in docker compose file. ( I dont want to harcode the password in compose file)

Added Entrypoint in docker-compose file and commented the original ALPINE password line below.
............
secrets:
  ldap:
    external: true

services:
  dtrack-apiserver:
    image: dependencytrack/apiserver
    secrets:
      - ldap
    entrypoint: sh -c "export ALPINE_LDAP_BIND_PASSWORD=$$(cat */ldap) && java $$JAVA_OPTIONS -DdependencyTrack.logging.level=$$LOGGING_LEVEL -jar $$WAR_FILENAME -context $$CONTEXT"

#- ALPINE_LDAP_BIND_PASSWORD=
.............

It says sync completed in log and there are no errors in logs as LDAP is configured but in DT Administration  -> Access Management, when i add any ldap user it says INVALID
<html>
<body>

</body>
</html>

If i do printenv inside the DT container api-server, all environment variables are there except password

**Comment by subbunulu** [comment]
Hi Steve,
i too am facing the same issue. Can you please help with correct configuration.
